sample/src/main/java/com/karumi/rosie/sample/base/view/activity/MarvelActivity.java

```diff
@@ -16,20 +16,29 @@
 
 package com.karumi.rosie.sample.base.view.activity;
 
-import android.support.annotation.Nullable;
 import android.support.design.widget.Snackbar;
 import android.view.View;
+
+import butterknife.ButterKnife;
 import butterknife.OnClick;
+import butterknife.Optional;
+
 import com.karumi.rosie.sample.R;
 import com.karumi.rosie.view.RosieActivity;
 
 public abstract class MarvelActivity extends RosieActivity {
   @Override protected abstract int getLayoutId();
 
-  @Nullable @OnClick(R.id.iv_toolbar_back) public void onBackButtonClicked() {
+  @Optional @OnClick(R.id.iv_toolbar_back) public void onBackButtonClicked() {
     finish();
   }
 
+  @Override
+  protected void onPreparePresenter() {
+    super.onPreparePresenter();
+    ButterKnife.bind(this);
+  }
```

Comment:
We should create another method invoked just after the setContentView method. Something like ``onPrepareActivity`` or just override ``setContentView`` method to add the butter knife call. Thoughts @Serchinastico?